src/decoder/stream.rs

```diff
@@ -449,7 +447,7 @@ impl From<TextDecodingError> for DecodingError {
 /// Decoder configuration options
 #[derive(Clone)]
 pub struct DecodeOptions {
-    ignore_adler32: bool,
+    pub(crate) ignore_adler32: bool,
```

Comment:
This is a bit ugly, might want to think about a better way to handle disable adler32 checksums now.